lib/DDG/Spice/Astrobin/Apod.pm

```diff
@@ -17,8 +17,10 @@ spice alt_to => {
 	}
 };
 
-handle remainder => sub {
+handle remainder => sub {        
+        my $query = lc($req->query_raw);        
         return '' if /^((image|photo|picture)s?)?\s*(of|of the)?\s*(day|today|daily)$/i;
+        return '' if ($query eq 'apod' and $_ eq '');
```

Comment:
@rasikapohankar you don't need the second check here `$_ eq ''` because `$query` is the whole query, meaning the trigger + remainder